RecoTracker/LSTCore/standalone/code/core/write_lst_ntuple.cc

```diff
@@ -579,6 +621,146 @@ void setPixelTripletOutputBranches(lst::Event<Acc3D>* event) {
   ana.tx->setBranch<std::vector<int>>("pT3_isDuplicate", pT3_isDuplicate);
 }
 
+//________________________________________________________________________________________________________________________________
+void fillT5DNNBranches(lst::Event<Acc3D>* event, unsigned int T3) {
```

Comment:
```suggestion
void fillT5DNNBranches(lst::Event<Acc3D>* event, unsigned int t3) {
```
or `iT3` to follow the naming style